labs/fabric-toolkit.md

```diff
@@ -0,0 +1,30 @@
+# Lab Name
+
+Hyperledger Fabric distributed deploy toolkit.
+
+# Short Description
+
+* This project is aimed to automate the deployment process of the hyperledger fabric.
+* The deployment tool provides basic mechanisms for deployment, repairment, and scaling of applications.
+
+# Scope of Lab
+
+This project is aimed to automate the deployment process of the hyperledger fabric. Hyperledger Fabric is a platform for distributed ledger solutions and the examples released can easily deploy the Hyperledger Fabric network in a standalone machine environment, including order, peer chaincode and so on.
+
+However, If you want to build up the Hyperledger Fabric network in the distributing development environments, it is not a easy task and you have to deploy the modules in each of the machine manually, which includes lots of configurations and commands and is inconvenient. Our preject is a tool to help you to install the hyperledger fabric in the distributing development environments easily and conveniently.
+
+The deployment tool provides basic mechanisms for deployment, repairment, and scaling of applications. Firstly, once you have completed the configuration of the orderer and peer，witch include the information of the role,the domain and ip address， the tool can deploy the orderer and the peer according the configuration you written by itself. Secondly, when the hyperledger fabric network is running， the tool can scale your hyperledger fabric network up and down with a simple command.
+
+# Initial Committers
+
+- [Ddpool0](https://github.com/Ddpool0) - Zheng Li
+- [rivulet2015](https://github.com/rivulet2015) - Yuejiang He
+- [zeoio](https://github.com/zeoio)
+
+# Sponsors
+
+- Credit Card Center of China Guangfa Bank (kinsleer@outlook.com)
```

Comment:
You need to have a qualifying sponsor. What makes your sponsor qualify?

Reply:
@zeoio is this PR actively maintained recently or stale?
@lehors Zeoio has looked for sponsorship within TWGC months ago, I was glad to be a sponsor. But later they hesitate on whether the project contains commercial sensitive information. blocking its contributing journey.